src/dataflow/components/dataflow-program.tsx

```diff
@@ -319,6 +319,13 @@ export class DataflowProgram extends BaseComponent<IProps, IState> {
             sensors.push(chInfo.channelId);
           }
         }
+      } else if (n.name === "Relay" && n.data.relayList) {
+        const chInfo = this.channels.find(ci => ci.channelId === n.data.relayList);
+        if (chInfo) {
+          if (hubs.indexOf(chInfo.hubId) === -1) {
+            hubs.push(chInfo.hubId);
+          }
+        }
```

Comment:
Is it correct that we do not need to get the relay id stored in `chInfo.channelId` and pass it to the program (as we do with the sensors)?

Reply:
We aren't actually using the sensor list at all. I could remove it, but there's a possible tiny efficiency we could add later by keeping it, so I'll leave `sensors` there. We have no need for `relays`, however.